ocdata/bulk_obj.py

```diff
@@ -14,8 +14,27 @@ class Bulk():
     '''
     This class handles all things with the bulk.
     It also provides possible surfaces, later used to create a Surface object.
+
```

Comment:
minor reorganization suggestion: could you reorganize the comment in the form of 
- one high level summary statement
- description of Attributes (class variables)
- description of methods (member functions)

An example from https://realpython.com/documenting-python-code/ is below:

```
    """
    A class used to represent an Animal

    ...

    Attributes
    ----------
    says_str : str
        a formatted string to print out what the animal says
    name : str
        the name of the animal
    sound : str
        the sound that the animal makes
    num_legs : int
        the number of legs the animal has (default 4)

    Methods
    -------
    says(sound=None)
        Prints the animals name and what sound it makes
    """
```